setup.py

```diff
@@ -8,5 +8,14 @@
     url="https://github.com/privML/privacy-evaluator",
     license="MIT",
     packages=find_packages(include=["privacy_evaluator*"]),
-    install_requires=["adversarial-robustness-toolbox[pytorch,tensorflow]==1.6.1"],
+    install_requires=[
+        "numpy",
+        "pandas",
+        "matplotlib",
+        "torch",
+        "torchvision",
+        "tensorflow",
+        "adversarial-robustness-toolbox[pytorch,tensorflow]==1.6.1",
```

Comment:
I think you can leave `[pytorch,tensorflow]` out here, since `pytorch` and `tensorflow` are already listed. Also it seems like it may cause problems while installing. I'm not sure if it's the cause but I randomly get 

```
ERROR: Could not find a version that satisfies the requirement adversarial-robustness-toolbox[pytorch,tensorflow]==1.6.1 (from privacy-evaluator)
ERROR: No matching distribution found for adversarial-robustness-toolbox[pytorch,tensorflow]==1.6.1
```

when trying to install in a notebook. What's weird is that sometimes the installation works and sometimes it doesn't.

Reply:
Might have been some kind of an issue with PyPi servers at the time as it works fine now.

Reply:
Yes, there was an issue with the PyPi servers. Also, all tests were failing at this time because `pytest` and `black` could not be installed. I also kept the [pytorch,tensorflow] is there because there might be also other dependencies listed for these "extras" (for PyTorch it is not the case at the moment but for TensorFlow `tensorflow_addons` and `h5py` ist also installed).